src/connection.ts

```diff
@@ -3071,6 +3071,23 @@ export class Connection {
     };
   }
 
+  /**
+   * Fetch confirmed blocks between two slots
+   */
+  async getBlocks(
+    startSlot: number,
+    endSlot?: number,
+    commitment?: Finality
+  ): Promise<Array<number>> {
+    const args = this._buildArgsAtLeastConfirmed([startSlot, endSlot], commitment);
```

Comment:
I don't think this will work when `endSlot` is not defined. Can you fix and add a test please?

Reply:
Do you mean that:

`endSlot ? [startSlot, endSlot] : [startSlot]`

Reply:
correct